elements/bulbs-liveblog/bulbs-liveblog-entry.js

```diff
@@ -16,6 +16,10 @@ class BulbsLiveblogEntry extends BulbsHTMLElement {
     if (this.liveblog && this.liveblog.handleEntryAttached) {
       this.liveblog.handleEntryAttached({ target: this });
     }
+
+    if (twttr) {
+      twttr.widgets.load();
+    }
```

Comment:
Perhaps we can note in the future a way to hook into an entry load finish so we can do this per site as needed.